epoch/api/chain_ws_api.md

```diff
@@ -133,7 +112,7 @@ Response:
   | Name | Type | Description | Required |
   | ---- | ---- | ----------- | -------- |
   | type | string | `header` | Yes |
-  | height | number | Height of the header to fetch | yes |
```

Comment:
It looks still present in https://github.com/aeternity/epoch/pull/1293 i.e. at https://github.com/aeternity/epoch/blob/9a09d588847ac5c37ab7bf834d8b15e26f45a5aa/apps/aehttp/src/ws_int_dispatch.erl#L34-L35 ?